### PR TITLE
[DOC] Fill installation section in readme

### PR DESCRIPTION
**Pull request by kuizhiqing**
Fill installation section in readme

### CODE REVIEW

**Review by GuoxiaWang — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -26,17 +26,27 @@ TODO: A GIF showing the tasks supported by PaddleFleetX.
...
 ## Installation
 
-### From Pip Install
+We recommend to get started with PaddleFleetX using [pre-build container](docs/quick_start.md#11-docker-环境部署) which comes with all requirements installed.
+If you prefer to install the requirements on your own, please following this installation guide.
+
+### Requirements
+
+* [PaddlePaddle](https://www.paddlepaddle.org.cn/) GPU version must be installed **before** using PaddleFleetX.
+* GPUs are required to work with PaddleFleetX, Nvidia V100 or above are recommended.
```
Comment (sneaxiy) [inline README.md]: Nvidia -> NVIDIA

File: README.md [hunk 2]
```diff
@@ -26,17 +26,27 @@ TODO: A GIF showing the tasks supported by PaddleFleetX.
...
+
+### Requirements
+
+* [PaddlePaddle](https://www.paddlepaddle.org.cn/) GPU version must be installed **before** using PaddleFleetX.
+* GPUs are required to work with PaddleFleetX, Nvidia V100 or above are recommended. 
+* Other PyPI requirements are listed in `requirements.txt`.
+
+### Install
+
+With PaddlePaddle well installed, you can fetch PaddleFleetX and install its dependents with the following commands,
```
Comment (sneaxiy) [inline README.md]: `install its dependents` -> `install its dependencies`.

File: README.md [hunk 3]
```diff
@@ -26,17 +26,27 @@ TODO: A GIF showing the tasks supported by PaddleFleetX.
 
 ## Installation
 
-### From Pip Install
+We recommend to get started with PaddleFleetX using [pre-build container](docs/quick_start.md#11-docker-环境部署) which comes with all requirements installed.
+If you prefer to install the requirements on your own, please following this installation guide.
```
Comment (sneaxiy) [inline README.md]: `prefer to installing`
`please follow`

**Pull request merged**